.github/workflows/build.yml

```diff
@@ -296,6 +302,25 @@ jobs:
           cp -R 'tmp.package/Assets/TextMesh Pro' Assets/
           rm -rf tmp.plugin tmp.package
 
+
+      - name: Manually delete Zapbox package
+        if: matrix.vrsdk != 'Zapbox'
+        run: |
+          # Android builds fail on if Zapbox package is present
```

Comment:
Unless we need it there (perhaps in the packages-lock file) for pinning versions, can we have optional packages not included and add them as needed? (I don't know if this is possible)

Reply:
See above, but in general unity's manifest.json is quite uncooperative

Reply:
ok, fair enough. I suppose this is also better for the package cache if they're all there.